src/sortWith.js

```diff
@@ -0,0 +1,45 @@
+var _curry2 = require('./internal/_curry2');
+var _slice = require('./internal/_slice');
+
+/**
+ * Sorts a list according to a list of comparators.
+ *
+ * @func
+ * @memberOf R
+ * @category Relation
+ * @sig [(a, a -> Number)] -> [a] -> [a]
```

Comment:
If we're taking an array of curried binary functions:

```haskell
sortWith :: [a -> a -> Number] -> [a] -> [a]
```

Otherwise:

```haskell
sortWith :: [(a, a) -> Number] -> [a] -> [a]
```